test/FunctionalTests/Microsoft.OData.Edm.Tests/ExtensionMethods/TargetHelperTests.cs

```diff
@@ -101,19 +101,19 @@ public void GetTargetSegmentsForEntitySetNavigationPropertyUnderDerivedComplexPr
         }
 
         [Fact]
-        public void GetTargetSegmentsForSingletonNavigationPropertyUnderComplexProperty()

```

Comment:
Rather than replace an existing test, can we add this as a new test?

Reply:
Sorry, that was just a goof - intended to add it as new.